cockroachcloud/optimize-serverless-workload.md

```diff
@@ -57,6 +63,38 @@ The amount of SQL CPU needed to execute this query is about 1.5 milliseconds. Th
 
 Note that this is not exact, as there can be slight variations in multiple parts of the calculation.
 
+You can use the [`EXPLAIN ANALYZE` SQL command](../{{site.versions["stable"]}}/explain-analyze.html) with your statements to estimate the RU usage of that statement. For example, the prepend `EXPLAIN ANALYZE` to the `INSERT` statement above:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+EXPLAIN ANALYZE INSERT INTO kv VALUES (1, '...imagine this is a 1 KiB string...');
+~~~
+
+~~~
+               info
+-----------------------------------
+  planning time: 13ms
+  execution time: 6ms
+  distribution: local
+  vectorized: true
+  maximum memory usage: 10 KiB
+  network usage: 0 B (0 messages)
+  estimated RUs consumed: 15
+
+  • insert fast path
+    nodes: n1
+    actual row count: 1
+    into: kv(k, v)
+    auto commit
+    size: 2 columns, 1 row
+(14 rows)
+
+
+Time: 71ms total (execution 20ms / network 50ms)
+~~~
+
+This will insert the data, but also output information from the optimizer about the execution of the statement, including the optimzer's estimate of RU consumption on the `estimated RUs consumed:` line. In this case, the optimizer estimated the `INSERT` statement would consume 15 RUs, which is similar to the estimate of 18.5 RUs we made earlier.
```

Comment:
```suggestion
This will insert the data, but also output information from the optimizer about the execution of the statement, including the optimizer's estimate of RU consumption on the `estimated RUs consumed:` line. In this case, the optimizer estimated the `INSERT` statement would consume 15 RUs, which is similar to the estimate of 18.5 RUs we made earlier.
```